addons/crm/tests/test_crm_pls.py

```diff
@@ -478,6 +478,47 @@ def test_predictive_lead_scoring(self):
         self.assertEqual(tools.float_compare(lead_tag_2.automated_probability, 28.6, 2), 0)
         self.assertEqual(tools.float_compare(lead_tag_1_2.automated_probability, 28.6, 2), 0)
 
+    def test_predictive_lead_scoring_always_won(self):
+        """ The computation may lead scores close to 100% (or 0%), we check that pending
+        leads are always in the ]0-100[ range."""
+        Lead = self.env['crm.lead']
+        LeadScoringFrequency = self.env['crm.lead.scoring.frequency']
+        country_ids = self.env['res.country'].search([], limit=3).ids
+        stage_ids = self.env['crm.stage'].search([], limit=3).ids
```

Comment:
No need to limit=3 here, we just use the first one. Let's do country_id and stage_id with `search([], limit=1).id`

Also I would argue it's better to directly reference a model we know exists instead of searching with hope but I see we already do this in these tests so fine.

Reply:
Yes, I copied the start from the test above. I'll adapt.